source/fundamentals/connection/tls.txt

```diff
@@ -66,6 +66,43 @@ using a method in the ``MongoClientSettings.Builder`` class.
                 .build();
          MongoClient client = MongoClients.create(settings);
 
+Configure Netty
+~~~~~~~~~~~~~~~
+
+If you use the driver with `Netty <https://netty.io/>`__ for network IO,
+you have an option to plug an alternative TLS/SSL protocol implementation
+provided by Netty.
+
+.. code-block:: java
+   :copyable: True
+
+   import com.mongodb.MongoClientSettings;
+   import com.mongodb.client.MongoClients;
+   import com.mongodb.client.MongoClient;
+   import com.mongodb.connection.netty.NettyStreamFactoryFactory;
+   import io.netty.handler.ssl.SslContext;
+   import io.netty.handler.ssl.SslContextBuilder;
+   import io.netty.handler.ssl.SslProvider;
+
+To instruct the driver to use `io.netty.handler.ssl.SslContext <https://netty.io/4.1/api/io/netty/handler/ssl/SslContext.html>`__,
+use the `NettyStreamFactoryFactory.Builder.sslContext <{+api+}/apidocs/mongodb-driver-core/com/mongodb/connection/netty/NettyStreamFactoryFactory.Builder.html#sslContext(io.netty.handler.ssl.SslContext)>`__
+method. See the documentation of this method for details on which `io.netty.handler.ssl.SslProviders <https://netty.io/4.1/api/io/netty/handler/ssl/SslProvider.html>`__
```

Comment:
```suggestion
method. See the method documentation for details on which `io.netty.handler.ssl.SslProviders <https://netty.io/4.1/api/io/netty/handler/ssl/SslProvider.html>`__
```
Personal preference/nitpick: trim "of this" that isn't needed to reduce word count.